src/course-updates/course-handouts/CourseHandouts.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from '@edx/paragon';
+import { useIntl } from '@edx/frontend-platform/i18n';
+
+import messages from './messages';
+
+const CourseHandouts = ({ contentForHandouts, onEdit, isDisabledButtons }) => {
+  const intl = useIntl();
+
+  return (
+    <div className="course-handouts" data-testid="course-handouts">
+      <div className="course-handouts-header">
```

Comment:
If we call this block a header in the class name, should it have an appropriate tag `<header>`?

Reply:
I think no